src/app/components/event-card/event-card.tsx

```diff
@@ -59,9 +60,9 @@ export const EventCard = ({
           <br />
           {city}
         </address>
-        <a href={href} target="_blank" rel="noreferrer" className={cx('details-button')}>
+        <LinkCustom href={href} button={true} size="small" target="_blank">
```

Comment:
We decided that this is not a button, but a link, because it has the href property. But it's styled as a button, so the button class is added. I can replace it with `className='button'`

Reply:
I got it and removed unnecessary code